tests/test_cleanup.py

```diff
@@ -0,0 +1,30 @@
+import os
+import urllib.request
+
+from finn.core.modelwrapper import ModelWrapper
+from qonnx.util.cleanup import cleanup
+
+
+def test_cleanup_cnv_w2a2():
+    # download test data
+    dl_dir = "/tmp"
+    dl_file = dl_dir + "/cnv-w2a2.onnx"
+    cnv_w2a2_qonnx_url = "https://drive.google.com/uc?id=1BAyYM3N28fDHSWMgK8dZ3kVUsxGZLO3U&export=download"
```

Comment:
I have changed the permissions on this file down from "can edit" to "can comment". In order to avoid the very unlikely scenario that someone on the internet grabs hold of this file and injects strange .onnx files into our test suite.

We should probably consider loading this from the qonnx model zoo in the future.